src/main/java/core/basesyntax/dao/FruitDaoImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReaderService;
+import core.basesyntax.service.WriterService;
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import core.basesyntax.service.impl.WriterServiceImpl;
+import java.util.List;
+import java.util.Map;
+
+public class FruitDaoImpl implements FruitDao {
+    private static final String FILE_NAME_INPUT_DATA = "src/main/resources/inputData.csv";
+    private static final String FILE_NAME_PROCESSED_DATA = "src/main/resources/outputData.csv";
```

Comment:
<img width="898" alt="image" src="https://user-images.githubusercontent.com/81781479/235314375-456efea9-61b7-443d-9557-1ac6d1eb5d2d.png">

you should pass this values into reader or writers